### DISCUSSION TITLE
post/react-router-dom-csr-prefetch/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#47] [Announcements]
# post/react-router-dom-csr-prefetch/

react-router-dom 기반의 CSR 환경에서 Suspense를 사용하면서 발생한 순차적 API호출 문제를 프리패칭을 통해 개선한 경험을 공유합니다.

https://tech.kakaopay.com/post/react-router-dom-csr-prefetch/

**Comment by hanyeonhee** [comment]
크으-! 민수님 너무 멋있어요! 항상 응원합니다

**Comment by alstn2468** [comment]
@hanyeonhee 아이고 코멘트까지 감사합니다 🙇‍♂️

**Comment by minsgy** [comment]
좋은 글 사례 공유 감사합니다!
